src/modules/data/entities/instance/instance.metagame.territory.entity.ts

```diff
@@ -70,14 +71,14 @@ export default class InstanceMetagameTerritoryEntity {
     })
     duration: number;
 
-    @ApiProperty({example: 1, enum: ps2alertsEventStateArray, description: 'The internal event state. 0 = scheduled, 1 = in progress, 2 = finished'})
+    @ApiProperty({example: Ps2alertsEventState.ENDED, enum: ps2alertsEventStateArray, description: 'The internal event state. 0 = scheduled, 1 = in progress, 2 = finished'})
     @Column({
         type: 'enum',
         enum: ps2alertsEventStateArray,
     })
     state: Ps2alertsEventState;
 
-    @ApiProperty({description: 'Victory data for the instance'})
+    @ApiProperty({example: {vs: 34, nc: 33, tr: 33, cutoff: 0, winner: Faction.VANU_SOVEREIGNTY, draw: false}, description: 'Victory data for the instance'})
```

Comment:
I think it's better to put the example in the embed class instead.

Reply:
3b410f2269ebc20d461ee4022d7e81f715fc642b